packages/vaadin-crud/src/interfaces.d.ts

```diff
@@ -95,15 +94,15 @@ export type CrudElementEventMap<T> = {
 
   'size-changed': CrudSizeChanged;
 
-  'new': CrudNew;
+  new: CrudNew;
```

Comment:
This looks suspicious, as the event name appears to use a reserved keyword `new`. 

As long as `tsc` passed, I take that as the fact that TypeScript does not really care.